jwt.js

```diff
@@ -0,0 +1,40 @@
+const config = require('./config')
+const debug = require('debug')('revshare-pointer:jwt')
+const jsonwebtoken = require('jsonwebtoken')
+
+const jwt = {
+  createToken: function(payload, options) {
+    options = addAlgorithm(options)
+    return jsonwebtoken.sign(payload, config.jwtSecret, options)
+  },
+
+  verifyToken: async function(token) {
+    let options = {
+      iss: config.jwtIssuer,
+      aud: config.jwtAudience,
+    }
+    options = addAlgorithm(options)
```

Comment:
addAlgorithm mutates the object passed in so we don't need this to be an assignment: the line can just be `addAlgorithm(options)`